### ISSUE TITLE
"Connection reset by peer" during "Set Kibana timestemp field" step

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#370]
Whey I tried bringing up a server on Google cloud I got this error:

```
TASK [Set Kibana timestamp field.] *********************************************
fatal: [104.154.128.194]: FAILED! => {"changed": false, "content": "", "failed": true, "msg": "Status code was not [200, 201]: Connection failure: [Errno 104] Connection reset by peer", "redirected": false, "status": -1, "url": "http://127.0.0.1:9200/.kibana/index-pattern/*"}
	to retry, use: --limit @/Users/s0198839/.loom/playbooks/gcloud_start_server.retry
```

I think we may be sending this request before Kibana is ready to receive requests. I ssh'd into the server and was able to "curl http://127.0.0.1:9200". I also tried re-running the "loom server start" command with the server from the failed command still up, and this time it succeeded. It seems like Kibana just needed a little more time, and polling might fix this.

In order to release 0.3.4 I removed the ansible steps to set kibana timestamp and default index, but let's add these back after debugging.

**Comment by isaacliao** [comment]
Makes sense. I'll add polling to make sure Kibana is ready before setting timestamp and index.

**Comment by isaacliao** [comment]
Implemented in 36973710a7de4b711f91fd72c02b5e0f2cac77ee.
